lib/Auth/Source/SP.php

```diff
@@ -131,6 +132,7 @@ public function getIdPMetadata($entityId) {
 			return $metadataHandler->getMetaDataConfig($entityId, 'shib13-idp-remote');
 		} catch (Exception $e) {
 			/* Metadata wasn't found. */
+            SimpleSAML\Logger::debug('getIdpMetadata: ' . $e->getMessage());
```

Comment:
Do you want to make the message logged for each of these distinct from each other some how? I'm not sure how important that is, but it seemed worth suggesting.